example/README.md

```diff
@@ -227,38 +229,38 @@ In the event that you are running the tests outside of the Cumulus sandbox envir
 
 ### Adding tests for an existing workflow
 
-Workflow tests are located in the `/spec/<workflow-name>` folder. Any tests and supporting JSON files can go in there.
+Workflow tests are located in the `/spec/parallel` or `/spec/serial` directory depending on whether or not they can run in parallel or not. Any tests and supporting JSON files can go in there.
 
 ### Adding a new test workflow
 
-The workflow should be configured as it would be for a normal Cumulus deployment in a workflows yaml file. It must be deployed to the current deployment if testing locally.
-
-The workflows yaml files are located in the `/workflows/` folder and are split up to make the workflows easier to find and understand. When adding a new file, make sure to update the `app/config.yml` `stepFunctions` field.
+The workflow should be configured as it would be for a normal Cumulus deployment in a workflows terraform file. It must be deployed to the current deployment if testing locally.
 
-A new folder should be added in the `/spec` folder for the workflow and the tests should go into that folder with the input JSON files.
+The workflows terraform files are located in the `/example/cumulus-tf` directory and are split up to make the workflows easier to find and understand.
 
-Ideally the test can run in parallel with other tests and should be put in the `parallel` folder. If it cannot be, it should go in the `spec` folder. Only if the test should be run outside of the test suite should it go in the `standalone` folder.
+A new directory should be added in either the `/spec/parallel` directory if the test workflow can be run in parallel. The tests should go in the newly created directory along with any necessary input JSON files. Otherwise, the new test workflow should be added to the `/spec/serial` directory. Only if the test should be run outside of the test suite should it go in the `standalone` directory.
```

Comment:
```suggestion
A new directory should be added in the `/spec/parallel` directory if the test workflow can be run in parallel. The tests should go in the newly created directory along with any necessary input JSON files. Otherwise, the new test workflow should be added to the `/spec/serial` directory. Only if the test should be run outside of the test suite should it go in the `standalone` directory.
```

Reply:
Ah, good catch. TY